pkg/cmd/target/command.go

```diff
@@ -9,6 +9,7 @@ package target
 import (
 	"errors"
 	"fmt"
+	drop "github.com/gardener/gardenctl-v2/pkg/cmd/target/drop"
```

Comment:
redundant `drop` alias.. will fix this in a new commit together with a fixed go import sorting